core/src/main/scala/kafka/tools/GetOffsetShell.scala

```diff
@@ -43,10 +43,11 @@ object GetOffsetShell {
                            .describedAs("partition ids")
                            .ofType(classOf[String])
                            .defaultsTo("")
-    val timeOpt = parser.accepts("time", "timestamp of the offsets before that")
+    val timeOpt = parser.accepts("time", "Timestamp of the offsets before that")
```

Comment:
NIT: may be it is better to be inline with other param descriptions, which does not start with caps.